static/app/views/alerts/list/rules/row.tsx

```diff
@@ -215,10 +215,10 @@ function RuleListRow({
       onAction: () => {
         openConfirmModal({
           onConfirm: () => onDelete(slug, rule),
-          header: t('Delete Alert Rule?'),
-          message: tct(
-            "Are you sure you want to delete [name]? You won't be able to view the history of this alert once it's deleted.",
-            {name: rule.name}
```

Comment:
Just an idea: I think it would be nice to visually distinguish the name in some way, like put it in quotes or italicize it or something